merge-v1/schema.graphql

```diff
@@ -25,30 +25,38 @@ An account of users holding an NFT within merge. collection.
 type User @entity {
   "The holder's address of NFT in collection"
   id: ID!
-  "Unique tokenID for a respective NFT"
-  tokenID: BigInt!
   "Details about NFTs being held by this User"
-  massNFTs: [NFTs!]! @derivedFrom(field: "owner") #pak: allows one to query info about NFTs belonging to this user
+  massNFTs: [NFT!] @derivedFrom(field: "owner") #pak: allows one to query info about NFTs belonging to this user
```

Comment:
`mergeNFTs` makes more sense to me too since it is the collection name. Each NFT within the collection is actually referred to as a `mass`. Considering the context of the client (front-end querying this subgraph), `mergeNFTs` make more sense to me since you're talking about the collection perspective.